src/content/docs/infrastructure/prometheus-integrations/integrations-list/node-exporter-integration.mdx

```diff
@@ -0,0 +1,418 @@
+---
+title: 'Node Exporter integration'
+tags:
+  - Integrations
+  - Node Exporter 
+  - Prometheus
+  - host monitoring 
+freshnessValidatedDate: 2024-04-10 
+---
+
+Do you want to monitor hardware and kernel metrics for a Linux server? You can do this with the New Relic remote write integration and the Prometheus Node Exporter. When you combine these two programs with the Prometheus monitoring system, you can send data to New Relic where you can analyze it. 
+
+## Prerequisites [#prerequisites]
+
+Our instructions here are based on the Prometheus guide [Monitoring Linux host metrics with the node exporter](https://prometheus.io/docs/guides/node-exporter/). We'll expand on those instructions with steps to help you send your data to New Relic.
```

Comment:
What if you said: "These instructions are based..."?

Reply:
I changed it to "The instructions here...."

Reply:
Like the change!